assets/search/components/SearchResultsBar/index.tsx

```diff
@@ -235,19 +236,26 @@ class SearchResultsBarComponent extends React.Component<IProps, IState> {
                                 >
                                     {gettext('Clear All')}
                                 </button>
-                                <button
-                                    data-test-id="toggle-search-bar"
-                                    onClick={this.toggleTagSection}
-                                    className="icon-button icon-button--tertiary icon-button--bordered"
-                                >
-                                    <i className={classNames(
-                                        'icon--arrow-right',
-                                        {
-                                            'icon--collapsible-open': isTagSectionShown,
-                                            'icon--collapsible-closed': !isTagSectionShown,
+                                <ToolTip key={`${isTagSectionShown}--state`} placement="left">
```

Comment:
I don't get the key here, is it supposed to be `true-state` / `false-state` ?

Reply:
yeah, just doing it because we have a dynamic `title` which has to be refreshed based on the state. No other logic behind the key